src/data/LedgerCache.cpp

```diff
@@ -1,7 +1,7 @@
 //------------------------------------------------------------------------------
 /*
     This file is part of clio: https://github.com/XRPLF/clio
-    Copyright (c) 2022, the clio developers.
+    Copyright (c) 2025, the clio developers.
```

Comment:
Why? If this file was created in 2022, it should be 2022.

Reply:
It looks weird with the interface created in 2025 😃 I can change back.

Reply:
I don't think it looks weird. 3 years ago the content of the file could be even completely different. But as I understand, the rule is to put the year of creation and never change it.